auth_token/models.py

```diff
@@ -36,6 +38,8 @@ class Token(models.Model):
     backend = models.CharField(max_length=255, null=False, blank=False)
     allowed_cookie = models.BooleanField(default=True)
     allowed_header = models.BooleanField(default=True)
+    is_two_factor_authenticated = models.BooleanField(null=False, blank=False, default=False)
+    two_factor_key = models.CharField(null=True, blank=True, max_length=128)
```

Comment:
key should be hashed. You use words code and key. It should be standardized (code will be better)